### ISSUE TITLE
CJS output isn't ES5

### ISSUE THREAD

**Issue by adierkens** [algolia/react-element-to-jsx-string#387]
The `/dist/cjs/index.js` file contains arrow functions which aren't commonjs

```
var lib = createCommonjsModule(function (module, exports) {
Object.defineProperty(exports, "__esModule", { value: true });
exports.default = (object) => Object
    .getOwnPropertySymbols(object)
    .filter((keySymbol) => object.propertyIsEnumerable(keySymbol));

});
```

This causes the project to break support on older browsers (like IE)

It looks like in `14.0.2` this worked: 
```
var lib = createCommonjsModule(function (module, exports) {
Object.defineProperty(exports, "__esModule", { value: true });
exports.default = function (object) { return Object
    .getOwnPropertySymbols(object)
    .filter(function (keySymbol) { return object.propertyIsEnumerable(keySymbol); }); };

});
```

**Comment by ahuth** [comment]
Seeing the same. Seems that this project's dependency `stringify-object` updated its own dependency on `get-own-enumerable-property-symbols` which packages an arrow function.

Perhaps this package can downgrade `stringify-object`?

**Comment by Spy-Seth** [comment]
We could yes. `stringify-object` is a pain point for us if it could be deleted it could be perfect

**Comment by hipstersmoothie** [comment]
Yeah we saw this too and is causing use to tweak out webpack configs to include this library

**Comment by Spy-Seth** [comment]
https://github.com/algolia/react-element-to-jsx-string/pull/408 is merged and release in [v4.1.0](https://github.com/algolia/react-element-to-jsx-string/releases/tag/v14.1.0) 🎉
